toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/Scalebar.kt

```diff
@@ -52,8 +63,21 @@ import kotlin.time.Duration.Companion.seconds
  * A Scalebar displays the representation of an accurate linear measurement on the map.
  * It provides a visual indication through which users can determine the size of features or
  * the distance between features on a map.
- * // TODO: update documentation
  *
+ * @param maxWidth the maximum screen width allotted to the scalebar in dp.
+ * @param unitsPerDip the number of map units per density-independent pixel (dp).
+ * @param viewpoint the current viewpoint of the map.
+ * @param spatialReference the spatial reference of the map.
+ * @param modifier the modifier to apply to this layout.
+ * @param autoHideDelay the duration to wait before hiding the scalebar, set to `Duration.INFINITE` to disable auto-hide.
+ * @param minScale the minimum scale to show the scalebar, default is `0.0` which means the scalebar will always be visible.
+ * @param useGeodeticCalculations `true` to compute scale using a geodesic curve, `false` to compute scale without a geodesic curve,
```

Comment:
This line could be shortened. 
```suggestion
 * @param useGeodeticCalculations `true` to compute scale using a geodesic curve, `false` otherwise.
```